src/main/frontend/mixins.cljs

```diff
@@ -127,10 +126,10 @@
      :did-mount (fn [state]
                   (attach-listeners state)
                   state)
-     :did-remount (fn [old-state new-state]
-                    (detach old-state)
-                    (attach-listeners new-state)
-                    new-state)
+     :will-remount (fn [old-state new-state]
```

Comment:
`will-remount` is not supported in the rum dep we're using, also, `will-remount` is equal to `did-remount` afaik, https://github.com/tonsky/rum/blob/gh-pages/src/rum/core.cljs#L45

Reply:
Yes, I will remove this commit.